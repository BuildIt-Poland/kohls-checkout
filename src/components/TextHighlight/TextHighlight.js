import styled from 'styled-components';
import Text from '../Text/Text';
import { colorGreen } from '../../styles/designTokens';

const TextHighlight = styled(Text)`
  color: ${colorGreen};
`;

export default TextHighlight;
