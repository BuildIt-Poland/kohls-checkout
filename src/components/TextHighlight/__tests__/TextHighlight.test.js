import React from 'react';
import { create } from 'react-test-renderer';

import TextHighlight from '../TextHighlight';

describe('COMPONENT - TextHighlight', () => {
  it('renders TextHighlight component', () => {
    const component = create(<TextHighlight />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
