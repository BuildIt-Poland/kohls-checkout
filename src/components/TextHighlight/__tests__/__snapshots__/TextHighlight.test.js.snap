// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - TextHighlight renders TextHighlight component 1`] = `
.c0 {
  font-size: 1.2rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: normal;
  color: #9cc472;
}

<span
  className="c0"
/>
`;
